.github/workflows/connect-dev-release-test.yml

```diff
@@ -8,8 +8,6 @@ on:
   schedule:
     # Runs at midnight UTC every day at 01:00 AM CET
     - cron: "0 0 * * *"
-  push:
-    branches: [npm-release/*, release/*]
```

Comment:
On `npm-release/*` it is not required since we already modify the files on those packages and it triggers the tests. what is happening now is that on that branch test are running 2 times, and it is not necessary.

I explained in the description of this PR why I remove the `release/*` from here.

Reply:
ok, so `release/*` is effectively replaced by the trigger script, right?

Reply:
Yes.